kythe/rust/indexer/tests/indexer_entries.rs

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 The Kythe Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::array_writer::ArrayWriter;
+
+extern crate kythe_rust_indexer;
+use kythe_rust_indexer::indexer::entries::EntryEmitter;
+use kythe_rust_indexer::error::KytheError;
+use storage_rust_proto::*;
+
+#[test]
```

Comment:
It would be nice to leave some documentation on what these tests are supposed to do.  In case someone else needs to resume this work, it will help them orient and understand what is expected of them to do.